app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({self.name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(
+        name=person["name"],
+        age=person["age"]
+    ) for person in people]
+
+    for person in people:
+        if person.get("wife") is not None:
+            Person.people[person["name"]].wife = \
+                Person.people[person["wife"]]
```

Comment:
Write inline or move Person.people[person["wife"]] to variable and then set
The same with husband